Dockerfile

```diff
@@ -6,13 +6,16 @@ ENV DEVICE_INDEX="" \
     FEED_ID="" \
     PPM="0"\
     GAIN="-10" \
-    RTLMULT="160" \
+    RATEMULT="160" \
```

Comment:
This will break existing installs if anyone has set this. I doubt they have, but you never know. Any reason it was changed?

Reply:
Fair enough, I think I made it match acarsdec since this imafe is no longer just rtlsdr. I'll change it back when I'm back at a computer.